script/input/1/base-staging.json

```diff
@@ -4,6 +4,7 @@
     "cctpTokenMessenger": "0x1682Ae6375C4E4A97e4B583BC394c861A46D8962",
     "psm": "0x1601843c5E9bC251A3272907010AFa41Fa18347E",
     "rateLimits": "0x79F826786953fb42aed02796F792EF8f2701d18b",
-    "safe": "0xaB959A6F88b8D966c44a7cDC2049Ba9669EBf047",
+    "relayer": "0x611C7c37F296240c2fF5a92f0B4a398B01B237c4",
+    "freezer": "0x611C7c37F296240c2fF5a92f0B4a398B01B237c4",
```

Comment:
Base staging safe is `0xaB959A6F88b8D966c44a7cDC2049Ba9669EBf047` - use this for both freezer and relayer